drivers/wdm/audio/backpln/portcls/irpstream.cpp

```diff
@@ -4,6 +4,7 @@
  * FILE:            drivers/wdm/audio/backpln/portcls/irpstream.cpp
  * PURPOSE:         IRP Stream handling
  * PROGRAMMER:      Johannes Anderwald
+ *                  Oleg Dubinskiy
```

Comment:
If you want to add yourself here, please format the headed according to our current coding style